pyproject.toml

```diff
@@ -48,7 +48,7 @@ dependencies = [
   "numpy >= 1.23.0",
   "packaging >= 22.0",
   "pandas >= 1.4.0",
-  "pytest == 7.4.4",
+  "pytest >= 7.0.0",
```

Comment:
`pytest` is one of our runtime dependencies because of (now private) test helper functionality, so it will be helpful to loosen the requirement for installation.